packages/shared/.env

```
@@ -1,3 +1,3 @@
-SANITY_API_DATASET="production"
+SANITY_API_DATASET="development"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)